tests/zfs-tests/tests/functional/inheritance/inherit_001_pos.ksh

```diff
@@ -394,28 +394,34 @@ if is_linux; then
 		"off" "" \
 		"visible" "off"
 else
```

Comment:
The `aclmode` and `aclinherit` properties don't exist under Linux since illumos and Linux took different approach to ACLs, NFS vs POSIX.  So they were removed from this test case and replaced in with `acltype` which is used by Linux.  We could refactor this to make it clear that only the ACL properties differ.